magefile.go

```diff
@@ -1718,23 +1718,37 @@ func createTestRunner(matrix bool, singleTest string, goTestFlags string, batche
 		ServiceTokenPath: serviceTokenPath,
 		Datacenter:       datacenter,
 	}
+
 	ogcProvisioner, err := ogc.NewProvisioner(ogcCfg)
 	if err != nil {
 		return nil, err
 	}
+
 	email, err := ogcCfg.ClientEmail()
 	if err != nil {
 		return nil, err
 	}
-	essProvisioner, err := ess.NewProvisioner(ess.ProvisionerConfig{
+
+	provisionCfg := ess.ProvisionerConfig{
 		Identifier: fmt.Sprintf("at-%s", strings.Replace(strings.Split(email, "@")[0], ".", "-", -1)),
 		APIKey:     essToken,
 		Region:     essRegion,
-	})
+	}
+
+	essProvisioner, err := ess.NewProvisioner(provisionCfg)
 	if err != nil {
 		return nil, err
 	}
 
+	provisionerMode := os.Getenv("TEST_INTEG_SERVERLESS")
+	if provisionerMode != "" {
+		fmt.Println(">>>> Using serverless ESS deployment")
```

Comment:
Should properly log this line for the selected stack provisioner every time, not just when serverless is selected.